### ISSUE TITLE
Cannot get pycryptodome 3.6.0 to work. missing _galois.pyd and _cpuid.pyd

### ISSUE THREAD

**Issue by CY-Qiu** [Nuitka/Nuitka#1249]
* Nuitka version, full Python version and Platform (Windows, OSX, Linux ...)

  > python -m nuitka --version
0.6.17.3
Commercial: None
Python: 3.4.4 (v3.4.4:737efcadf5a6, Dec 20 2015, 19:28:18) [MSC v.1600 32 bit (Intel)]
Executable: C:\Users\xxx\.virtualenvs\sample_server-5Lg4ytw4\Scripts\python.exe
OS: Windows
Arch: x86
* How did you install Nuitka and Python
pip in a virtualenv

* The specific PyPI names and versions

  It should be taken from this output if there specific packages involved, e.g.
  `numpy`, you are expected to shorten this to the relevant ones.

  > python -m pip freeze
pycryptodome 3.6.0

* Also supply a [Short, Self Contained, Correct, Example](https://sscce.org/)
  code:  python -m --follow-imports --standalone --follow-imports --show-progress --include-package=Crypto.Util --include-package=Crypto.Cipher.AES
  Result exe fail to start: OSError: Cannot load native module 'Crypto.Util._galois': Trying '_galois.pyd': [WinError 126] 找不到指定的模块。
 After I manually add _galois.pyd into Crypto\\Util\\, another problem happend.
Result exe fail to start: OSError: Cannot load native module 'Crypto.Util._cpuid': Trying '_cpuid.pyd': [WinError 126] 找不到指定的模块。
After I manually add _cpuid.pyd into Crypto\\Util\\, the exe finally can boot successfully.

* Note if this is a regression
 No

**Comment by kayhayen** [comment]
A minimal reproducer is needed me to solve this. A command line is not that.

**Comment by kayhayen** [comment]
Closing for lack of feedback.
